src/main/java/com/marklogic/appdeployer/AppConfig.java

```diff
@@ -153,17 +153,17 @@
     // Comma-delimited string of database names that should only have forests (most likely just one) created on one host
     private Set<String> databasesWithForestsOnOneHost;
 
-    private Map<String, Set<String>> databaseHosts;
-    private Map<String, Set<String>> databaseGroups;
+    private Map<String, List<String>> databaseHosts;
```

Comment:
Tangential change - for consistency, made this into Lists instead of Sets. Was using a LinkedHashSet under the hood, but I think it's more expected for a user for these to be Lists, because there's an ordering to e.g. hosts.